cmake/pkgconfig.in

```diff
@@ -8,4 +8,4 @@ Version: v@OSRM_VERSION_MAJOR@.@OSRM_VERSION_MINOR@.@OSRM_VERSION_PATCH@
 Requires:
 Libs: -L${libdir} -losrm
 Libs.private: @ENGINE_LIBRARY_LISTING@
-Cflags: -I${includedir} -I${includedir}/osrm
+Cflags: -I${includedir} -I${includedir}/osrm -I@CMAKE_INCLUDE_PATH@ @CMAKE_CXX_FLAGS@
```

Comment:
@TheMarex We need some way to reflect the include paths of dependencies, when those dependencies are not installed into the same prefix as osrm. Do you have a better idea of how to do that?